tests/dynamical/test_state_trajectory.py

```diff
@@ -1,23 +0,0 @@
-import logging
```

Comment:
Minor nit: I think this test should be removed in a subsequent PR that merges `Trajectory` and `State`.